app/shared/design-system/components/button.tsx

```diff
@@ -160,6 +161,13 @@ export const Button = styled("button", {
             "inset 0 0 0 1px hsla(0,100%,100%,.35), 0 0 0 1px hsla(0,100%,100%,.35)",
         },
       },
+      raw: {
```

Comment:
why not just using Avatar component from design system?

Reply:
it has interactive prop

Reply:
I think this button shuld exist either way but added the avatar component from radix instead